functions/functions.go

```diff
@@ -114,6 +114,9 @@ func (f *Functions) DeleteFunction(id FunctionID) error {
 	if err != nil {
 		return &ErrorNotFound{id}
 	}
+
+	f.Log.Info("function deleted", zap.String("functionId", string(id)))
```

Comment:
For the message. Hmm I think that every other log message starts with capital letter. If not it has to be fixed.